htdocs/takepos/invoice.php

```diff
@@ -439,11 +439,14 @@ function fail($message)
         else $sectionwithinvoicelink.=$langs->trans('BillShortStatusValidated');
     }
     $sectionwithinvoicelink.='</span>';
+    if ($conf->global->TAKEPOS_AUTO_PRINT_TICKETS) {
```

Comment:
Something seems wrong here. There is more { opened then closed.
@andreubisquerra Can you help me fix this (what should be into if ($conf->global->TAKEPOS_AUTO_PRINT_TICKETS) and what should not ?

Reply:
For me the correct line is:
if ($conf->global->TAKEPOS_AUTO_PRINT_TICKETS) $sectionwithinvoicelink.='<script language="javascript">$("#buttonprint").click();</script>';
instead of
if ($conf->global->TAKEPOS_AUTO_PRINT_TICKETS) {

Reply:
@andreubisquerra change done